### PR TITLE
pick_ik: 1.0.0-1 in 'rolling/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by tylerjw**
Increasing version of package(s) in repository `pick_ik` to `1.0.0-1`:

- upstream repository: https://github.com/PickNikRobotics/pick_ik.git
- release repository: https://github.com/PickNikRobotics/pick_ik-release.git
- distro file: `rolling/distribution.yaml`
- bloom version: `0.11.2`
- previous version for package: `null`

## pick_ik

```
* pick_ik inverse kinematics plugin compatible with MoveIt 2
* Numeric gradient descent (local) solver
* Memetic algorithm (global solver), configurable for single or multi-threading
* Basic goal functions: joint centering, avoid joint limits, minimal joint displacement
* Support for position-only IK and approximate solutions
* Dynamic parameter switching at runtime using generate_parameter_library <https://github.com/PickNikRobotics/generate_parameter_library>
* Docker devcontainer workflow for VSCode
* Contributors: Chris Thrasher, Sebastian Castro, Tyler Weaver
```

### CODE REVIEW

File: rolling/distribution.yaml
Comment (hidmic) [inline rolling/distribution.yaml:3071]: @tylerjw IIRC proper Rolling release requires a release repository under the `ros2-gbp` organization. You may want to do the same for the Humble release (https://github.com/ros/rosdistro/pull/35592), just for the sake of convenience. CC @nuclearsandwich.

Comment (tylerjw) [reply rolling/distribution.yaml:3071]: I'm happy to put this in a different release repo. I was not aware of this rule. Let me know if you'd like me to move this?

Comment (hidmic) [reply rolling/distribution.yaml:3071]: @tylerjw yes please. See https://www.ros.org/reps/rep-2002.html#managing-build-farm-writable-release-repositories for the rationale and check https://docs.ros.org/en/rolling/How-To-Guides/Releasing/Release-Team-Repository.html on joining existing release teams to get your `ros2-gbp` release repo.

Comment (tylerjw) [reply rolling/distribution.yaml:3071]: I created this issue to have new release repos created for my three current projects that are not part of ros2-gbp and have them moved to the moveit release team: https://github.com/ros2-gbp/ros2-gbp-github-org/issues/163

Comment (tylerjw) [reply rolling/distribution.yaml:3071]: @hidmic Friendly ping on this. I create the issue over on ros2-gbp to request the release repos be created so I can migrate over there. Can you approve my request and create the new repos?

**Comment by tfoote** [comment]
Thanks for opening the PR to migrate the release files. 

I merged #35592 for humble but will hold this one as rolling needs the ros2-gbp repos for us to prep for Iron shortly. 

https://github.com/ros2-gbp/ros2-gbp-github-org/issues/163 looks correct and we'll process that early in the new year when we're back from vacation.

**Review by methylDragon — APPROVED** [review]
[Holding for rolling sync](https://discourse.ros.org/t/preparing-for-rolling-sync-2023-01-06/29052)

**Pull request merged**
